java/src/main/java/com/powsybl/dataframe/network/NetworkDataframes.java

```diff
@@ -958,11 +961,13 @@ static NetworkDataframeMapper branches() {
                 .stringsIndex("id", Branch::getId)
                 .strings("type", branch -> branch.getType().toString())
                 .strings("voltage_level1_id", branch -> branch.getTerminal1().getVoltageLevel().getId())
+                .strings("bus_breaker_bus1_id", branch -> getBusBreakerViewBusId(branch.getTerminal1()), (branch, id) -> setBusBreakerViewBusId(branch.getTerminal1(), id), false)
```

Comment:
should it be also in other branche dataframes like lines or twt ?

Reply:
It is already